src/bentoml/_internal/cloud/schemas/schemasv2.py

```diff
@@ -112,9 +112,9 @@ class LogSchema:
 @attr.define
 class LogWSResponseSchema:
     __forbid_extra_keys__ = False
-    message: str
+    message: str | None
```

Comment:
```suggestion
    message: t.Optional[str]
```

Reply:
‌‌‌‌‌‌‌Why do we use `T | None` in some parts of our code and `t.Optional[str]` in others? What rules should I follow?

<img width="1153" alt="image" src="https://github.com/bentoml/BentoML/assets/1206493/bdfca826-2ae9-47da-8861-f732272c2539">

Reply:
When it needs to be (de)serialized, it has to be Optional[...] since the `T|None` was not parsable at **runtime** on Python<=3.9

Reply:
As expected, `list[T]` doesn't work in Python 3.8 either. Facing such a divided situation is somewhat mentally taxing. Should we specifically perform static type checks for the schemas directory in CI for Python 3.8?